drivers/SmartThings/matter-window-covering/src/init.lua

```diff
@@ -22,6 +22,14 @@ local MatterDriver = require "st.matter.driver"
 local DEFAULT_LEVEL = 0
 local PROFILE_MATCHED = "__profile_matched"
 
+local OperationalStateEventEnum = {}
+OperationalStateEventEnum.NO_EVENT = 0x00
+OperationalStateEventEnum.CURRENT_POSITION_EVENT = 0x01
+OperationalStateEventEnum.OPERATIONAL_STATE_EVENT = 0x02
```

Comment:
```suggestion
local OperationalStateEventEnum = {
  NO_EVENT = 0x00
  CURRENT_POSITION_EVENT = 0x01
  OPERATIONAL_STATE_EVENT = 0x02
}
```

Reply:
Alternatively, I believe you could be using `st.matter.clusters.OperationalState.types.OperationalStateEnum` here, which has values for `STOPPED`, `RUNNING`, `PAUSED`, and `ERROR`

Reply:
Thank you for your recommend. I think OperationStateEventEnum should be changed to EventEnum.